### DISCUSSION TITLE
How to AddCalendar using ASP.NET Core integration NuGet?

### DISCUSSION THREAD

**Discussion by prcdpr** [quartznet/quartznet#945] [General] [answered]
Previously using bare Quartz.Net I could do something like this:

```
await scheduler.AddCalendar(nameof(MyCustomCalendar), calendar, true, true);
```

Is it possible with new `Quartz.AspNetCore` API?

**Comment by lahma** [comment, marked as answer]
I did a PR to include this in 3.2, does the API fulfill your needs?

https://github.com/quartznet/quartznet/pull/946/files?short_path=6c242b7#diff-6c242b79007199a71ed1f6c1ff5aac93

**Comment by prcdpr** [comment]
Wow, thanks for swift PR. Yes, it does fulfill my needs, exactly what I've been missing.

**Reply by lahma** [reply]
Great,  you can get it from MyGet preview feed as soon as the build for master completes.

**Reply by lahma** [reply]
Quartz 3.2 has been released.

**Reply by prcdpr** [reply]
That's great news, thank you
